doc/release/release_dev.rst

```diff
@@ -19,6 +19,13 @@ Highlights
 This release is the result of X of work with over X pull requests by
 X contributors. Highlights include:
 
+Warning: Hash values observed in outputs of `weisfeiler_lehman_graph_hash` 
+have changed from version 2.6 -> 2.7 onwards, due to bug fixes 
+[discussed here](https://github.com/networkx/networkx/pull/4946#issuecomment-914623654). 
+This means that comparing graph hashes of hashes before and after version 2.7 
+could wrongly fail an isomorphism test (isomorphicgraphs always have matching 
+Weisfeiler-Lehman hashes). Users are advised to recalculate any stored graph 
+hashes they may have on upgrading.
```

Comment:
@rossbar @dschult added a draft of a warning to `release_dev.rst` on hash changes. Feel free to rewrite this. I think I've resolved all other issues in the PR.